Add info about the grafana deployer action and api key (#929)
2i2c-org/infrastructure: docs/howto/operate/grafana.md

**Keep this key safe as you won't be able to retrieve it!**

Encrypt and store this key using `sops` in `secrets/config/hubs/<cluster>.yaml` under `grafana_token` key.

> This key will be used by the [`deploy-grafana-dashboards` Github Action](https://github.com/2i2c-org/infrastructure/blob/HEAD/.github/workflows/deploy-grafana-dashboards.yaml) to deploy some default grafana dashboards for JupyterHub using[`jupyterhub/grafana-dashboards`](https://github.com/jupyterhub/grafana-dashboards).

Reviewer: ```suggestion
This key will be used by the [`deploy-grafana-dashboards` Github Action](https://github.com/2i2c-org/infrastructure/blob/HEAD/.github/workflows/deploy-grafana-dashboards.yaml) to deploy some default grafana dashboards for JupyterHub using [`jupyterhub/grafana-dashboards`](https://github.com/jupyterhub/grafana-dashboards).
```

> This key will be used by the [`deploy-grafana-dashboards` Github Action](https://github.com/2i2c-org/infrastructure/blob/HEAD/.github/workflows/deploy-grafana-dashboards.yaml) to deploy some default grafana dashboards for JupyterHub using [`jupyterhub/grafana-dashboards`](https://github.com/jupyterhub/grafana-dashboards).

Reviewer 2: ```suggestion
This key will be used by the [`deploy-grafana-dashboards` workflow](https://github.com/2i2c-org/infrastructure/blob/HEAD/.github/workflows/deploy-grafana-dashboards.yaml) to deploy some default grafana dashboards for JupyterHub using [`jupyterhub/grafana-dashboards`](https://github.com/jupyterhub/grafana-dashboards).
```

> Once you've pushed the ecrypted `grafana_token` to the GitHub repository, manually trigger the GitHub Action to deploy the dashboads.

Reviewer: ```suggestion
Once you've pushed the encrypted `grafana_token` to the GitHub repository, manually trigger the GitHub Action to deploy the dashboards.
```

> Once you've pushed the encrypted `grafana_token` to the GitHub repository, manually trigger the GitHub Action to deploy the dashboards.

Reviewer 2: ```suggestion
Once you've pushed the encrypted `grafana_token` to the GitHub repository, manually trigger the `deploy-grafana-dashboards` workflow using the "Run workflow" button [from here](https://github.com/2i2c-org/infrastructure/actions/workflows/deploy-grafana-dashboards.yaml) to deploy the dashboards.
```

```{note}

> The action only runs when manually triggered.

Reviewer 2: ```suggestion
The workflow only runs when manually triggered.
```

> Any re-triggering of the action after the initial deployment will overwrite any dashboard created from the Grafana UI and not stored in the [`jupyterhub/grafana-dashboards`](https://github.com/jupyterhub/grafana-dashboards) repository.

Reviewer 2: ```suggestion
Any re-triggering of the workflow after the initial deployment will overwrite any dashboard created from the Grafana UI and not stored in the [`jupyterhub/grafana-dashboards`](https://github.com/jupyterhub/grafana-dashboards) repository.
```
